docs/ref/models/querysets.txt

```diff
@@ -2244,7 +2244,10 @@ them, but it has a few caveats:
   ``batch_size``.
 * Updating fields defined on multi-table inheritance ancestors will incur an
   extra query per ancestor.
-* If ``objs`` contains duplicates, only the first one is updated.
+* When ``objs`` contains instances with equal values of ``pk``, only the first 
+  such instance will affect the resulting query. If (due to ``batch_size``)
+  multiple queries are required, only the first instance per batch will have
+  an effect.
```

Comment:
IMO we don't need to be so detailed, it's really niche. Maybe :thinking: :
```
* If ``objs`` contains duplicates, only the first one is updated, unless
  multiple queries are required and duplicates are in a different batches.
```

Reply:
I agree it should probably be more succinct. Two small thoughts though:
 - Is it right to talk about items in `objs` as being updated? Since the instances themselves are not updated - rather they affect an update to the db.
 - In the case of duplicates split between batches, should we explicitly say what does happen, rather than saying what doesn't happen (which leaves open other possibilities)

How about:
```
* When an individual batch contains duplicates, only the first
  such instance in that batch will result in an update.
```

Reply:
Sounds good, I would chop `such`.